docs/MIGRATION.md

```diff
@@ -1,5 +1,118 @@
+  <details>
+    <summary>Adding Dashboard Provider, Dashboards and plugins in Prometheus-stack</summary>
+
+    ### Adding Dashboard Provider in Prometheus-stack
+    * Dashboard Provider should be added in grafana by dashboardProviders.dashboardproviders.yaml.
+    ```
+      grafana:
+        dashboardProviders:
+          dashboardproviders.yaml:
+            apiVersion: 1
+            providers:
+              - name: 'default'
+                orgId: 1
+                folder: ''
+                type: file
+                disableDeletion: false
+                editable: true
+                options:
+                  path: /var/lib/grafana/dashboards/default
+    ```
+    ### Adding Dashboard Urls in Prometheus-stack
+    * Dashboards should be added in grafana from infra-repo by Url.
+    ```
+      grafana:
+        dashboards:
+          default:
+            Cassandra-dashboard:
+              url: http://infra-repo.service.svc.cluster.local:8080/dashboards/cassandra-dashboard_rev2.json
+            Rabbitmq-dashboard:
+              url: http://infra-repo.service.svc.cluster.local:8080/dashboards/rabbitmq-overview_rev11.json
+            Node-monitoring:
+              url: http://infra-repo.service.svc.cluster.local:8080/dashboards/nodes-monitoring-v1.0.0.json
+            Namespace-health:
+              url: http://infra-repo.service.svc.cluster.local:8080/dashboards/namespace_health-v1.0.0.json
+            Components-logs:
+              url: http://infra-repo.service.svc.cluster.local:8080/dashboards/components-logs.json
+            Monitoring-1:
+              url: http://infra-repo.service.svc.cluster.local:8080/dashboards/Monitoring-1632997642909.json
+            Monitoring-2:
+              url: http://infra-repo.service.svc.cluster.local:8080/dashboards/Monitoring-1636451927534.json
+            Monitoring-3:
+              url: http://infra-repo.service.svc.cluster.local:8080/dashboards/Monitoring-1648485494141.json
+    ``` 
+    ### Adding Plugin Urls in Prometheus-stack
+    * Plugins should be added in grafana from infra-repo by plugins.
+    ```
+      grafana:
+        plugins:
+          - http://infra-repo.service.svc.cluster.local:8080/plugins/flant-statusmap-panel-0.4.2.zip;flant-statusmap-panel
+          - http://infra-repo.service.svc.cluster.local:8080/plugins/vonage-status-panel-1.0.11.zip;vonage-status-panel
+          - http://infra-repo.service.svc.cluster.local:8080/plugins/blackmirror1-statusbygroup-panel-1.1.2.zip;blackmirror1-statusbygroup-panel
+          - http://infra-repo.service.svc.cluster.local:8080/plugins/grafana-polystat-panel-1.2.8.any.zip;grafana-polystat-panel
+          - http://infra-repo.service.svc.cluster.local:8080/plugins/briangann-gauge-panel-0.0.9.zip;briangann-gauge-panel
+          - http://infra-repo.service.svc.cluster.local:8080/plugins/yesoreyeram-boomtable-panel-1.4.1.zip;yesoreyeram-boomtable-panel
+    ```
+  </details>
+* Rabbitmq values should be updated.
```

Comment:
Please add warning note that previous rabbitmq persistence data must be deleted before installation